src/python/library/tritonclient/utils/__init__.py

```diff
@@ -125,6 +125,7 @@ def debug_details(self):
 
 
 def np_to_triton_dtype(np_dtype):
+    from bfloat16 import bfloat16
```

Comment:
Not asking for change here, but just a note for future reference - if we ever encounter any issues with importing this package - we can defer the import to only be when it is actually needed by placing this in the `else` block. That way, no one will ever hit the bf16 import unless they're using a type not already covered by this method.

I think this is OK how it is for now though.

i.e.:
```
if np_dtype == bool:
  ...
# If not covered by builtin numpy types, defer custom imports here to only be when needed
else:
    from bfloat16 import bfloat16
    if np_dtype == bfloat16:
        return "BF16"

return None
```